src/components/validation/runner/telemetry.rs

```diff
@@ -145,7 +145,6 @@ impl Telemetry {
             }
 
             grpc_task_coordinator.shutdown().await;
-            debug!("GRPC task(s) have been shutdown.");
```

Comment:
This is perhaps a nit, but: why remove this?

By calling `shutdown`, you _are_ ensuring that the tasks that registered themselves with this coordinator -- so long as they're marking themselves as started and complete at the right points -- have in fact shutdown, since it does both the shutdown trigger and waits for all children to complete.

Reply:
Ah, I removed this because I think it's redundant with the changes I made to `TaskCoordinator` in this PR- in the shutdown() method (called the line above), we log an `info` level message that says the task has been shutdown.